src/main/sensors/voltage.h

```diff
@@ -54,7 +55,7 @@ typedef enum {
 #define VBAT_SCALE_MIN 0
 #define VBAT_SCALE_MAX 255
 
-#define VBATT_LPF_FREQ  1.0f
```

Comment:
With the changes it takes about .5s to move 1V.  If the faster filtered is need somewhere and unfiltered can't be used, I could add another voltage filter variable.